requirements.txt

```diff
@@ -27,6 +27,7 @@ scandir==1.9.0
 six==1.11.0
 smmap2==2.0.5
 stevedore==1.21.0
+tenacity==5.0.3
```

Comment:
This needs to be added to the setup.py file too.

Reply:
Fixed !